Simplified/Colors.xcassets/buttonBackground.colorset/Contents.json

```diff
@@ -5,9 +5,9 @@
         "color-space" : "srgb",
         "components" : {
           "alpha" : "1.000",
-          "blue" : "0xFE",
-          "green" : "0xFC",
-          "red" : "0xFC"
+          "blue" : "0.996",
```

Comment:
some of these values are not hex value because I didn't have the hex values in the Figma, so I had to detect it via the Digital color meter.

Some other values somehow Xcode changed to floating point for some reason.